header/p2p/subscriber.go

```diff
@@ -12,14 +12,14 @@ import (
 	"github.com/celestiaorg/celestia-node/header"
 )
 
-// Subscriber manages the lifecycle and relationship of header Service
```

Comment:
There are bunch of decapitalization in changes and as thosenl international or relic from some global renaming?

Reply:
Yes, must have been from refactoring in GoLand

Reply:
Replaced now where it makes sense, there are still some usages of lowercase service inside the package that are correct

Reply:
👌🏻